src/frontends/pytorch/include/openvino/frontend/pytorch/node_context.hpp

```diff
@@ -86,7 +86,12 @@ class NodeContext : public frontend::NodeContext {
     // TODO: upstream to base class
     OutputVector inputs() const {
         OutputVector res;
```

Comment:
test for the fix?

Reply:
I'll add it roman. This is a unique case where the error i get is only from models and not single node tests. I might have to fix some unit tests as well which are breaking

Reply:
move implementation from header to source file?

Reply:
Done